lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -75,6 +99,10 @@ def data_dir_empty?
       false
     end
 
+    def relabel_postgresql_dir
```

Comment:
Do we need this here? The data directory should be empty before we do the clone.